dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/TracingInterceptor.kt

```diff
@@ -246,7 +246,7 @@ internal constructor(
             .asChildOf(parentContext)
             .start()
 
-        (span as? MutableSpan)?.resourceName = url
+        (span as? MutableSpan)?.resourceName = url.substringBefore(URL_QUERY_PARAMS_BLOCK_SEPARATOR)
```

Comment:
@xgouchet should we also do something about automatically created spans ? Those are automatically created on the backend so I am thinking that they could apply this fix there right ?

Reply:
The ones created in the backend already follow the rule